ppr-ui/src/assets/styles/overrides.scss

```diff
@@ -420,30 +415,28 @@ tbody {
 }
 
 .column-selection .v-list .v-list-item__title {
-  font-size: 14px !important;
   color: $gray7;
+  font-size: 0.875rem !important;
 }
 
 .column-selection .v-list .v-icon {
-  font-size: 20px !important;
+  color: $gray7;
+  font-size: 1rem !important;
 }
 
 .column-selection .v-list-item__action:first-child {
-  margin-right: 10px;
-  margin-top: 0px;
-  margin-bottom: 0px;
+  margin: 0 10px 0 0;
 }
 
 .column-selection .v-input__slot {
   height: 45px;
-  padding-left: 10px;
-  padding-top: 4px;
+  padding-left: 12px;
 }
 
 .column-selection .v-input__icon .v-icon {
-  padding-top: 8px;
-  padding-right: 8px;
-  color: $primary-blue;
```

Comment:
are these not supposed to be blue?

Reply:
No, they asked for grey in the new invision design